CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Replace intermediate confirmation step in rollover protocol for the maker with a configurable flag which can be updated during runtime using the new `POST /rollover/config` endpoint.
   On startup, rollovers are configured to be accepted by default.
+- Introduce the `/itchysats/order/1.0.0` protocol, running over libp2p.
```

Comment:
Mhmm... Since you wrote `introduce` I wanted to say this should be under `Added` but actually we are `replacing bespoke network stack` with this... sooo... happy to leave it where it is 🙃